gibbon-compiler/solve_ilp_-34.py

```diff
@@ -0,0 +1,22 @@
+from docplex.mp.model import Model
```

Comment:
This PR adds three py-files inside `gibbon-compiler`: I don't think they should be there. they seem to be leftovers from the layout work which is not related to this PR?..